datafusion/core/Cargo.toml

```diff
@@ -67,7 +67,9 @@ datafusion-physical-expr = { path = "../physical-expr", version = "8.0.0" }
 datafusion-row = { path = "../row", version = "8.0.0" }
 datafusion-sql = { path = "../sql", version = "8.0.0" }
 futures = "0.3"
+glob = "0.3.0"
 hashbrown = { version = "0.12", features = ["raw"] }
+itertools = "0.10"
```

Comment:
This is technically a new dependency of this crate, but it is so ubiquitous and is a dependency of prost, etc.. so this is probably ok

Reply:
But datafusion doesn't depend on prost, right? If we could avoid another dependency that would be goo in my opinion

Reply:
Well once we bring in the new object_store we will depend on it...

Ultimately it is such a fundamental crate, like bytes, parking_lot, etc... I'm inclined to think it isn't really a problem...

Reply:
ok, I still cringe a little, but I vote :shipit: